data/traces/example/dirigent.csv

```diff
@@ -1,5 +1,5 @@
-HashFunction,Image,Port,Protocol,ScalingUpperBound,ScalingLowerBound,IterationMultiplier
-c13acdc7567b225971cef2416a3a2b03c8a4d8d154df48afe75834e2f5c59ddf,empty,80,tcp,5,0,80
-a2faad786b3c813b12ce57d349d5e62f6d0f22ceecfa86cd72a962853383b600,empty,80,tcp,5,0,80
-7dc5aeabc131669912e8c793c8925cc9928321f45f13a4af031592b4611630d7,empty,80,tcp,5,0,80
-ae8a1640fa932024f59b38a0b001808b5c64612bd60c6f3eb80ba9461ba2d091,empty,80,tcp,5,0,80
\ No newline at end of file
+HashFunction,Image,Port,Protocol,ScalingUpperBound,ScalingLowerBound,IterationMultiplier,IOPercentage
+c13acdc7567b225971cef2416a3a2b03c8a4d8d154df48afe75834e2f5c59ddf,docker.io/cvetkovic/dirigent_empty_function:latest,80,tcp,5,0,102,0
```

Comment:
If I understand correctly, function hash is not globally unique. We need to use all three (owner, app and func) to identify the function.